server/services/store/sqlstore/sqlstore.go

```diff
@@ -49,7 +51,14 @@ func New(params Params) (*SQLStore, error) {
 		pluginAPI:        params.PluginAPI,
 	}
 
-	err := store.Migrate()
+	var err error
+	store.isBinaryParam, err = store.computeBinaryParam()
+	if err != nil {
+		params.Logger.Error(`Cannot compute binary parameter`, mlog.Err(err))
+		return nil, err
+	}
+
+	err = store.Migrate()
```

Comment:
What happens if the binary flag is added to an existing database?

Reply:
AFAIK, the `binary_parameters` flag controls the way the driver communicates with the DB, but does not affect the final contents of the table, so it should work correctly.

Tested locally to confirm, but pinging @agnivade in case I'm missing anything.

Reply:
Yep, what Miguel said. `binary_parameters` just determines the client driver protocol. How postgres actuall stores the data is always the same.